### PR TITLE
feat(application-estate): Settlement Entry Requirements

### PR DESCRIPTION
**Pull request by kksteini**
* Rejection messages added based on validation
* Client Config updated
* Client Config generation updated

image

Note: District Commissioner's have stopped updating the `/dev` path and the newer client configs for development shall be retrieved from `/dev`
## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Formatting passes locally with my changes
- [X] I have rebased against main before asking for a review

### CODE REVIEW

**Review by RunarVestmann — APPROVED** [review]

File: libs/application/templates/estate/src/forms/OverviewSections/assetsAndDebts.ts [hunk 1]
```diff
@@ -51,11 +51,7 @@ export const overviewAssetsAndDebts = [
               (asset.marketValue ? formatCurrency(asset.marketValue) : '0 kr.'),
             m.propertyShare.defaultMessage +
               ': ' +
-              (asset.share
-                ? asset.share > 1
-                  ? asset.share + '%'
-                  : asset.share * 100 + '%'
-                : '0%'),
+              (asset.share ? asset.share + '%' : ''),
```
Comment (HelgiHH) [inline libs/application/templates/estate/src/forms/OverviewSections/assetsAndDebts.ts:54]: some of these might cause conflicts with https://github.com/island-is/island.is/pull/13055

File: libs/application/template-api-modules/src/lib/modules/templates/estate/estate.service.ts [hunk 2]
```diff
@@ -69,6 +69,36 @@ export class EstateTemplateService extends BaseTemplateApiService {
       )
     }
 
+    const { availableSettlements } = applicationData
+    console.log('Application answers', JSON.stringify(applicationAnswers))
```
Comment (HelgiHH) [inline libs/application/template-api-modules/src/lib/modules/templates/estate/estate.service.ts:73]: this console is intentional right?

**Review by HelgiHH — APPROVED** [review]
lgtm, might want to check on 2 comments/suggestions

File: libs/application/template-api-modules/src/lib/modules/templates/estate/estate.service.ts (hunk 2 above)
Comment (kksteini) [reply libs/application/template-api-modules/src/lib/modules/templates/estate/estate.service.ts:73]: Haha nope

Comment (kksteini) [reply libs/application/template-api-modules/src/lib/modules/templates/estate/estate.service.ts]: Fixed, thank you

File: libs/application/templates/estate/src/forms/OverviewSections/assetsAndDebts.ts (hunk 1 above)
Comment (kksteini) [reply libs/application/templates/estate/src/forms/OverviewSections/assetsAndDebts.ts:54]: The mappers have been rewritten to give the original value in percentages.
Is there something I'm missing here?

The value of share will now always be as a percentage, no [0,1] ratios

**Review by obmagnusson — APPROVED** [review]

**Pull request merged**